doc/changes/names.inc

```diff
@@ -266,6 +266,10 @@
 
 .. _Robert Luke: https://github.com/rob-luke
 
+.. _Robert Seymour: https://neurofractal.github.io
+
+.. _Matt Sanderson: https://www.linkedin.com.au/in/matt-sanderson-0a3667180
```

Comment:
@monkeyman192 I want to credit your original work on the code here (shared with me by Paul Sowman), is this a good URL to use for you?

Reply:
(If it's not good for you, feel free to make a quick PR to my branch or "suggest" in GitHub and I'm happy to incorporate. This also has the advantage of adding your name/email in the commit history which is nice.)

Reply:
Woo! Glad this code is finally going to good use!
Umm, maybe using my github page would be more consistent here. I'll make a suggestion, thanks!

Reply:
If you made a suggestion it did not show up, maybe you forgot to click "comment" or "Finish Review" or something?

Reply:
Sorry! I thought I had put it through. It appears I did in fact forget to press 'finish review'!

Reply:
Yes, I have done this for several days and also completely lost my suggestions before so you're not alone!